src/app/app.module.ts

```diff
@@ -49,12 +41,30 @@ const routes: Routes = [
   imports: [
     BrowserModule,
     HttpClientModule,
-    OAuthModule.forRoot(),
     FormsModule,
     ReactiveFormsModule,
     FontAwesomeModule,
     AppRoutingModule,
-    RouterModule.forRoot(routes),
+    AuthModule.forRoot({
```

Comment:
i believe you should move this to a module of its own and return an array of configs. It'll be less of a breaking change later on https://nice-hill-002425310.azurestaticapps.net/docs/documentation/configuration#using-multiple-configs

Reply:
Migrated to its own separate auth module